### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by babakfp** [tailwindlabs/tailwindcss#12262] [Help] [answered]
https://play.tailwindcss.com/6uE6wmJEZm?file=config

**Comment by wongjn** [comment, marked as answer]
If I am interpreting your intentions correctly, you'd want to use a function for the `height` configuration, so that you have access to `theme()` and thus, can reference `spacing.header`: https://play.tailwindcss.com/GEFbmXf8As?file=config
